src/main/java/domain/AttendanceStatus.java

```diff
@@ -0,0 +1,23 @@
+package domain;
+
+import java.time.LocalTime;
+import java.time.temporal.ChronoUnit;
+
+public enum AttendanceStatus {
+    
+    출석,
+    지각,
+    결석,
+    ;
+    
+    private static final int MAX_ATTEND_ADMIT_MINUTE = 5;
+    private static final int MAX_LATE_ADMIT_MINUTE = 30;
+    
+    public static AttendanceStatus of(LocalTime attendTime, LocalTime targetAttendTime) {
+        var minuteDifference = ChronoUnit.MINUTES.between(targetAttendTime, attendTime);
```

Comment:
```suggestion
    public static AttendanceStatus of(LocalTime targetAttendTime, LocalTime attendTime) {
        var minuteDifference = ChronoUnit.MINUTES.between(targetAttendTime, attendTime);
```
사소한 가독성이지만 파라메터의 순서와 밑에서 쓰이는 순서가 동일해야 코드를 읽는데 부담이 덜합니다.

Reply:
직접 코드를 다시 읽어보니 확실히 후자가 훨씬 더 잘 읽히는 것 같습니다! 감사합니다!
[refactor: 파라미터의 순서를 가독성 좋게 배치](https://github.com/woowacourse/java-attendance/pull/19/commits/2347ef9f6eab77d90f3f50c1022fb962bfd07887)